docs/tutor.txt

```diff
@@ -0,0 +1,226 @@
+=================================================================
+
+ Use :w/:write to save a file.
+
+ 1. Exit Helix using :q! as explained before, or open a new
+    terminal.
+ 2. Open a file in Helix by running: hx FILENAME
+ 3. Make some edits to the file.
+ 4. Press the : key to enter command mode.
+ 5. Type w or write, and press <ENTER> to save the file.
+
+ You can also use wq or write-quit to save and exit.
+
+ Note: You can optionally enter a filepath after the w/write
+       command in order to save to that path.
+ Note: If there are any unsaved changes to a file, a plus [+]
+       will appear next to the file name in the status bar.
+
+
+
+
+=================================================================
+=                             RECAP                             =
+=================================================================
+
+ * Use the h,j,k,l keys to move the cursor.
+
+ * Press : to enter command mode.
+   * The q/quit and q!/quit! commands will exit Helix. The
+     former fails when there are unsaved changes. The latter
+     discards them.
+   * The w/write command will save the file.
+   * The wq/write-quit command will do both.
+
+ * Press d to delete the character at the cursor.
+
+ * Press i to enter Insert mode and type text. Press <ESC> to
+   return to Normal mode.
+
+
+
+
+
+
+=================================================================
+=                    MOTIONS AND SELECTIONS                     =
+=================================================================
+
+ Press w to select forward until the next word.
+
+ The d key doesn't actually delete the character at the cursor,
+ it deletes all selected text. Your cursor is like a
+ single-character selection.
+
+ 1. Move the cursor to the line below marked -->.
+ 2. Move to the beginning of a word that needs to be deleted.
+ 3. Press w to select until the beginning of the next word.
+ 4. Press d to delete the selection.
+ 5. Repeat for all extra words in the line.
+
+ --> This sentence pencil has vacuum extra words in the it.
+
+
+
+
+
+
+=================================================================
+=                        MORE ON MOTIONS                        =
+=================================================================
+
+ As you saw, pressing w moves the cursor forward until the start
+ of the next word, selecting the text traversed. This is useful
+ for moving around text and for selecting text to operate on.
+
+ Some common motions include:
+   w - Move forward to before the beginning of the next word.
+   e - Move forward to the end of the current word.
+   b - Move backward to the beginning of the current word.
+
+ The w,e,b motions also have counterparts - W,E,B - which
+ traverse WORDS instead of words. WORDS are only separated by
+ whitespace, whereas words can be separated by other characters
+ in addition to whitespace.
+
+ All of these motions select the text they traverse.
```

Comment:
I think we should have a section discussing the multiple selections at least.

Reply:
I am planning to add a section introducing multiple selections; the tutorial is not complete yet.